src/createWidgetBase.ts

```diff
@@ -160,11 +151,8 @@ const createWidget: WidgetBaseFactory = createEvented
 				return properties;
 			},
 
-			classes: [],
-
 			getNode(this: Widget<WidgetProperties>): DNode {
-				const tag = formatTagNameAndClasses(this.tagName, this.classes);
-				return v(tag, this.getNodeAttributes(), this.getChildrenNodes());
+				return v('div', {}, this.getChildrenNodes());
```

Comment:
By switching to a default of `div`, don't we essentially make this unusable for dataviz where we may want to create widgets as SVG? Or would a div always wrap an svg tag and then an svg dom?